apps/hoarder/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+services:
+  web:
+    image: ghcr.io/hoarder-app/hoarder:0.16.0
+    restart: unless-stopped
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+    ports:
+      - "8685:3000"
+    environment:
+      MEILI_ADDR: http://meilisearch:7700
+      BROWSER_WEB_URL: http://chrome:9222
```

Comment:
For the changes I requested in lines `43` and `54` make sure you change it here too